### ISSUE TITLE
Server restart every day

### ISSUE THREAD

**Issue by L3o-pold** [ONLYOFFICE/Docker-DocumentServer#219]
Every day in the `/var/log/supervisor/supervisord.log` I see that the server restart.
Using `onlyoffice/documentserver-de:5.4.1.39`

```
2020-01-27 06:35:54,187 WARN received SIGTERM indicating exit request
2020-01-27 06:35:54,188 INFO waiting for metrics, spellchecker, gc, docservice, converter to die
2020-01-27 06:35:54,190 INFO stopped: spellchecker (terminated by SIGTERM)
2020-01-27 06:35:54,190 INFO stopped: metrics (exit status 0)
2020-01-27 06:35:54,191 INFO stopped: converter (terminated by SIGTERM)
2020-01-27 06:35:54,195 INFO stopped: docservice (terminated by SIGTERM)
2020-01-27 06:35:54,234 INFO stopped: gc (terminated by SIGTERM)
2020-01-27 06:35:59,271 CRIT Supervisor running as root (no user in config file)
```

```
2020-01-26 06:32:48,147 WARN received SIGTERM indicating exit request
```

```
2020-01-25 06:40:51,297 WARN received SIGTERM indicating exit request
```

`cat /etc/apt/apt.conf.d/20auto-upgrades`

```
APT::Periodic::Update-Package-Lists "0";
APT::Periodic::Unattended-Upgrade "0";
```

Any idea?

**Comment by ShockwaveNN** [comment]
Does server restart at same time? Does your system have enought ram and swap?

**Comment by L3o-pold** [comment]
It restart every day around 6AM. Yeah enough RAM and swap, seems more like an automatic shutdown but I don't see why.

**Comment by ShockwaveNN** [comment]
So it's just a supervisor restart? Container doesn't restart itself?

**Comment by L3o-pold** [comment]
Exactly.

**Comment by ShockwaveNN** [comment]
Maybe remove `unattended-upgrades` altogether to rule it out? I have no other idea (

**Comment by L3o-pold** [comment]
Should be reproductible if you use the docker image, once a day.
I'll try to solve this but if you can reproduce and/or have any other idea, could be nice.

**Comment by ShockwaveNN** [comment]
Seems this is `logrotate` server restarts
You can modify settings in `/etc/logrotate.d/ds.conf` or disable by removing this file

**Comment by L3o-pold** [comment]
Nice find. I'll give it a try.

**Comment by ShockwaveNN** [comment]
@L3o-pold I'm still not sure what restarting server by `logrotate` is good idea on our side, but currently we'll not change it

**Comment by L3o-pold** [comment]
@ShockwaveNN Yeah I'm not sure reloading the server, is a good idea, cause it'll break connections...
Could you send me a link where this logrotate is configured in the docker image or server repo please?

**Comment by ShockwaveNN** [comment]
> Could you send me a link where this logrotate is configured in the docker image or server repo please?

I'm not sure what do you mean. It configure in `/etc/logrotate.d/ds.conf` file as I mention earlier

**Comment by L3o-pold** [comment]
How is this file created?

**Comment by ShockwaveNN** [comment]
`/etc/logrotate.d/ds.conf` created while installing deb
It's located at https://github.com/ONLYOFFICE/document-server-package/blob/release/v5.5.0/common/documentserver/logrotate/ds.conf.m4 but sorry, this repo is private (

**Comment by L3o-pold** [comment]
Ok that why I can't find it. Feel free to close this (I don't recommend it because I don't see how nobody haven't see this issue) or not.

**Comment by ShockwaveNN** [comment]
@L3o-pold I'll leave issue open, because I'm not sure that it's a good idea to restart service daily, but currently do not how to elegant solve this problem
